templates/pdb.yaml

```diff
@@ -12,5 +12,5 @@ spec:
   maxUnavailable: {{ .Values.agent.pdb.maxUnavailable }}
 {{- end }}
   selector:
-    matchLabels: {{- toYaml .Values.agent.matchLabels | indent 6 }}
+    matchLabels: {{- toYaml .Values.agent.matchLabels | nindent 6 }}
```

Comment:
Thanks for the contribution! I discussed this with the team and the feedback was to move the {{ }} to its own line. Otherwise I think this looks good!

Reply:
hi @ ryanwohra , thanks for the feedback . TBH , i did not understand this phrase "move the {{ }} to its own line" . i just changed the spelling because it is breaking helm charts when enabling pdb  . Please let me know what else is expected . i will try to change it . thanks ,

Reply:
I merged in your requested change @ryanwohara